tests/e2e/e2e_pod_client_server_test.go

```diff
@@ -166,7 +178,7 @@ func setupSource(sourceName string, withKubernetesService bool) *v1.Pod {
 	}
 
 	// Expect it to be up and running in it's receiver namespace
-	Expect(Td.WaitForPodsRunningReady(sourceName, 90*time.Second, 1, nil)).To(Succeed())
+	Expect(Td.WaitForPodsRunningReady(sourceName, 240*time.Second, 1, nil)).To(Succeed())
```

Comment:
240 timeout seems unusually large. Could you make the timeouts specific to Windows so we can detect perf regressions for Linux.

Reply:
Windows images are considerably larger so downloading them on a fresh cluster takes some time if they are not cached. Sure will do.